src/features/user/Account/components/AccountRecord.tsx

```diff
@@ -80,9 +91,7 @@ export default function AccountRecord({
               <TransferDetails account={record.details.account} />
             )}
             {showStatusNote(record, t)}
-            {(record?.details?.donorCertificate ||
-              record?.details?.taxDeductibleReceipt ||
-              record?.details?.giftCertificate) && (
+            {showCertificate && record?.purpose !== 'conservation' && (
```

Comment:
This logic will hide all certificates for conservation projects, including tax certificates, gift certificates etc. 

Is that the desired outcome, or should only donor certificate links be not shown?

Reply:
@mohitb35 , @sunilsabatp only the donor certificate link should not be shown.